From 882d842df9822a6ee68dd8f2cabcdf1cc9898445 Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Tue, 2 Jul 2019 00:35:20 +0000
Subject: [PATCH] TOOLS-2101 jsstyle submodules too old for new perls Reviewed
 by: Brian Bennett <brian.bennett@joyent.com> Approved by: Brian Bennett
 <brian.bennett@joyent.com>

---
 .gitmodules         | 4 ++--
 deps/javascriptlint | 2 +-
 deps/jsstyle        | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/.gitmodules b/.gitmodules
index d88ef71..abaa1df 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,9 +1,9 @@
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
-	url = https://github.com/davepacheco/javascriptlint
+	url = https://github.com/joyent/javascriptlint
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = https://github.com/davepacheco/jsstyle
+	url = https://github.com/joyent/jsstyle
 [submodule "deps/restdown"]
 	path = deps/restdown
 	url = https://github.com/trentm/restdown
diff --git a/deps/javascriptlint b/deps/javascriptlint
index 040bf5e..ad52812 160000
--- a/deps/javascriptlint
+++ b/deps/javascriptlint
@@ -1 +1 @@
-Subproject commit 040bf5e429969ae42d976b570fd64c9e17fab20f
+Subproject commit ad52812e77bdfb1e90fb71a1201adb2b665a27e6
diff --git a/deps/jsstyle b/deps/jsstyle
index 194dde7..52dc973 160000
--- a/deps/jsstyle
+++ b/deps/jsstyle
@@ -1 +1 @@
-Subproject commit 194dde793f58b707b0416a106e1f7679795e50df
+Subproject commit 52dc973cf64da11834eca7cf46ebce8518e3ee88
-- 
2.21.0

